lib/bokken/uploaders/avatar.ex

```diff
@@ -7,13 +7,23 @@ defmodule Bokken.Uploaders.Avatar do
 
   @versions [:original, :thumb]
   @extension_whitelist ~w(.jpg .jpeg .gif .png)
+  @max_file_size 50_000_000
 
-  # Whitelist file extensions:
   def validate({file, _}) do
+    size = file_size(file)
+
     file.file_name
     |> Path.extname()
     |> String.downcase()
-    |> then(&Enum.member?(@extension_whitelist, &1))
+    |> then(&Enum.member?(@extension_whitelist, &1)) and check_file_size(size)
+  end
+
+  defp check_file_size(size) do
+    if size > @max_file_size do
+      false
+    else
+      true
+    end
```

Comment:
You can remove this function and simply do `and size > @max_file_size` above. I think it's prettier and saves a function call.